project.clj

```diff
@@ -9,7 +9,8 @@
                  [ring "1.3.2"]
                  [compojure "1.3.3"]
                  [clj-time "0.9.0"]
-                 [org.clojure/data.json "0.2.6"]]
+                 [org.clojure/data.json "0.2.6"]
+                 [org.clojure/tools.cli "0.3.1"]]
```

Comment:
I can, but this was the most recent version, when I opened this request :-). Merge and then update the Version?